components/awards/ScoreBasedAwards.js

```diff
@@ -113,6 +117,23 @@ export default function Performances({ award }) {
     }
   }
 
+  async function deleteAward() {
+    setLoading(true);
+    try {
+      await axios({
+        method: 'PUT',
```

Comment:
POST?

Reply:
I think we call `PUT` everywhere else, but I agree, it should be `POST`